leshan-client-core/src/main/java/org/eclipse/leshan/client/servers/ServersInfoExtractor.java

```diff
@@ -102,9 +102,13 @@ public static ServersInfo getInfo(Map<Integer, LwM2mObjectEnabler> objectEnabler
 
                         boolean useOscore = oscoreObjLink.getObjectId() == OSCORE;
                         if (useOscore) {
+                            LOG.trace("Bootstrap connection is using OSCORE.");
```

Comment:
Maybe something like  : 
```suggestion
                            LOG.trace("Server {} is using OSCORE.", info);
```
if we add this trace should we also add the same kind of trace for PSK, RPK, X509.
(or we just remove this trace)